src/components/sidebar/sidebar.spec.js

```diff
@@ -68,10 +90,24 @@ describe('Sidebar', () => {
     describe('clicking the close icon sidebar', () => {
       it('closes the sidebar', () => {
         const icon = wrapper.find('.carbon-sidebar__close-icon');
-        icon.simulate('click');
+        icon.at(0).simulate('click');
         expect(spy).toHaveBeenCalled();
       });
     });
+
+    describe('pressing Tab key on the close icon sidebar', () => {
+      it('closes the sidebar', () => {
+        const icon = wrapper.find('.carbon-sidebar__close-icon');
+        icon.at(0).props().onKeyDown({ which: 13, preventDefault });
+        expect(spy).toHaveBeenCalled();
+      });
+
+      it('does not close the sidebar', () => {
```

Comment:
The describe says `pressing Tab key on the close icon sidebar` then there are two contradicting `it` blocks. Should it close or not?